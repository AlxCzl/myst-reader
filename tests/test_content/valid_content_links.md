---
title: "Valid Content with Fictitious Paths"
author: "My Author"
date: "2020-10-16"
---
This is some valid content that should pass. If it does not pass we will know something is wrong.

Our fictitious internal files are available [here]({filename}/path/to/file):

Our fictitious static files are available [here]({static}/path/to/file):

Our fictitious attachments are available [here]({attach}path/to/file):
