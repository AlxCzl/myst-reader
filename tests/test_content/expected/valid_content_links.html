<p>
 This is some valid content that should pass. If it does not pass we will know something is wrong.
</p>



<p>
 Our fictitious internal files are available
 <a class="reference external" href="{filename}/path/to/file">
  here
 </a>
 :
</p>



<p>
 Our fictitious static files are available
 <a class="reference external" href="{static}/path/to/file">
  here
 </a>
 :
</p>



<p>
 Our fictitious attachments are available
 <a class="reference external" href="{attach}path/to/file">
  here
 </a>
 :
</p>
